inference-contract/docs/notebooks/templates/token-issuer.py

```diff
@@ -0,0 +1,232 @@
+# that interact with one another. By default, the context file used in this notebook 
+# is specific to the token. If you prefer to use a common context file, edit the 
+# `context_file` variable below.
+#
+# For the most part, no other modifications should be required.
+
+# %%
+token_path = 'token.' + token_class
+context_file = bindings.expand('${etc}/${token_class}_context.toml')
+print("using context file {}".format(context_file))
+
+context_bindings = {
+    'asset_type.identity' : token_owner,
+    'vetting.identity' : token_owner,
+    'guardian.identity' : token_owner,
+    'token_issuer.identity' : token_owner,
+    'token_issuer.count' : count,
+    'token_issuer.description' : token_description,
+    'token_issuer.token_metadata.opaque' : token_metadata,
+    'token_object.identity' : token_owner,
+    'guardian.url' : 'http://' + service_host + ':7900'
+}
+
+context = pc_jupyter.ml_jupyter.initialize_token_context(state, bindings, context_file, token_path, **context_bindings)
+pc_jupyter.pbuilder.Context.SaveContextFile(state, context_file, prefix=token_path)
+print('context initialized')
+
+
+# %% [markdown]
+# ### Create the Token Issuer Contract
+#
+# The process of creating the token issuer will also create an asset type contract object, 
+# a vetting organization contract object, and the guardian contract object. 
+# The asset type and vetting organization contract objects are principally used 
+# to complete the canonical asset interface that enables transparent value exchanges 
+# with tokens and other digital assets.
+
+# %%
+token_issuer_context = pc_jupyter.pbuilder.Context(state, token_path + '.token_issuer')
+token_issuer_save_file = token_issuer_context.get('save_file')
+if not token_issuer_save_file :
+    token_issuer_save_file = pc_jupyter.pcommand.invoke_contract_cmd(
+        pc_jupyter.ex_token_issuer.cmd_create_token_issuer, state, token_issuer_context)
+    pc_jupyter.pbuilder.Context.SaveContextFile(state, context_file, prefix=token_path)
+print('token issuer contract in {}'.format(token_issuer_save_file))
+
+
+# %% [markdown]
+# <hr style="border:2px solid gray">
+#
+# ## Operate on the Contract
+
+# %% [markdown]
+# ### Mint the Tokens
+
+# %%
+token_object_context = pc_jupyter.pbuilder.Context(state, token_path + '.token_object')
+
+minted_token_save_files = pc_jupyter.pcommand.invoke_contract_cmd(
+    pc_jupyter.ml_inference_token_object.cmd_mint_tokens, state, token_object_context)
+pc_jupyter.pbuilder.Context.SaveContextFile(state, context_file, prefix=token_path)
+
+minted_token_contexts = []
+for token_index in range(1, len(minted_token_save_files)+1) :
+    minted_token_contexts += [ token_object_context.get_context('token_{}'.format(token_index)) ]
+
+print("{} tokens minted".format(len(minted_token_save_files)))
+
+
+
+
+# %% [markdown]
+# ### Create Token Notebooks
+#
+# Create a token notebook for each of the minted tokens.
+
+# %%
+# %%skip True
```

Comment:
I know these (undocumented!) skips are inherited but i still found them quite annoying ...

Reply:
actually it is documented. exchange/pdo/contracts/jupyter.py (this is CODE so the question is whether code documents it).  and also in docs/notebooks/README.md